spirv_msl.cpp

```diff
@@ -5707,29 +6496,6 @@ void CompilerMSL::emit_barrier(uint32_t id_exe_scope, uint32_t id_mem_scope, uin
 			bar_stmt += "mem_flags::mem_none";
 	}
 
-	if (msl_options.is_ios() && (msl_options.supports_msl_version(2) && !msl_options.supports_msl_version(2, 1)))
```

Comment:
Is this okay to remove?

Reply:
Another accidental removal during the rebase. Sorry for that.